packages/wallets/src/components/Base/Divider/Divider.tsx

```diff
@@ -2,12 +2,13 @@ import React from 'react';
 import './Divider.scss';
 
 type TProps = {
+    color?: React.CSSProperties['color'];
     margin?: React.CSSProperties['margin'];
     variant?: 'horizontal' | 'vertical';
 };
 
-const Divider: React.FC<TProps> = ({ margin, variant = 'horizontal' }) => (
-    <div className={`wallets-divider__${variant}`} style={{ margin }} />
+const Divider: React.FC<TProps> = ({ color, margin, variant = 'horizontal' }) => (
+    <div className={`wallets-divider__${variant}`} style={{ color, margin }} />
 );
```

Comment:
I believe doing this would not work as the divider only contains a 1px solid border, so we'll have to use borderColor instead of color. You can do something like:

```
type TProps = {
    color?: React.CSSProperties['borderColor'];
    margin?: React.CSSProperties['margin'];
    variant?: 'horizontal' | 'vertical';
};

const Divider: React.FC<TProps> = ({ color = '#f2f3f4', margin, variant = 'horizontal' }) => (
    <div className={`wallets-divider__${variant}`} style={{ color, margin }} />
);
```

The default color in the Divider.scss file should also be removed.

Reply:
Done